source/_integrations/worldclock.markdown

```diff
@@ -19,37 +20,15 @@ related:
 
 The `worldclock` {% term integration %} simply displays the current time in a different time zone.
 
-## Configuration
+{% include integrations/config_flow.md %}
 
-To enable this {% term integration %} in your installation, add the following to your {% term "`configuration.yaml`" %} file.
-{% include integrations/restart_ha_after_config_inclusion.md %}
-
-```yaml
-# Example configuration.yaml entry
-sensor:
-  - platform: worldclock
-    time_zone: America/New_York
-```
-
-{% configuration %}
-time_zone:
-  description: The resource or endpoint that contains the value.
-  required: true
-  type: string
-name:
+{% configuration_basic %}
+Timezone:
+  description: Select a valid timezone from the list.
+Name:
   description: The name of the sensor, e.g., the city.
-  required: false
-  type: string
-  default: Worldclock Sensor
-time_format:
-  description: The time format.
-  required: false
-  type: string
-  default: "%H:%M"
-{% endconfiguration %}
-
-For valid time zones, check the **TZ** column in the [Wikipedia overview](https://en.wikipedia.org/wiki/List_of_tz_database_time_zones), or get the full list from the [pytz](https://pypi.python.org/pypi/pytz) module.
+Time format:
+  description: The time format, defaults to "%H:%M" which represents hour and minute.
+{% endconfiguration_basic %}
 
-```shell
-python3 -c "import pytz;print(pytz.all_timezones)"
-```
+For valid time zones, check the **TZ** column in the [Wikipedia overview](https://en.wikipedia.org/wiki/List_of_tz_database_time_zones).
```

Comment:
The update above suggests, that the tz should be selected from a list, making this potentially unnecessary?

Reply:
There is a list in the config flow but I kept this as a reference for users to lookup timezones in relation to countries etc.
A very long plain list in a config flow can be a bit hard to find what you're searching for.

Reply:
Sounds good to me! I'm not sure if the select supports typing/filtering by typing, but that might fix the need for this in the future.